app/controllers/vote_controller.rb

```diff
@@ -23,8 +23,7 @@ def cast_vote
     pass_query_params
     naming = Naming.find(params[:id].to_s)
     observation = naming.observation
-    value = params[:value].to_i
-    observation.change_vote(naming, value)
+    observation.change_vote(naming, params[:value])
     redirect_with_query(controller: :observer,
                         action: :show_observation,
                         id: observation.id)
```

Comment:
There's a conflict (see below), which should be resolved as follows:

```suggestion
    observation = naming.observation
    observation.change_vote(naming, params[:value])
    redirect_to observation_path(observation.id, q: get_query_param)
```

@mo-nathan: What's the best practice for resolving conflicts for PR's that simply port `master` changes to `bootstrap-4`
Is it:
(a) suggest modifications to the PR?
(b) close the PR, merge `master` into `bootstrap-4`, fix conflicts, create a new PR, merge the new PR?
(c) something else?
```ruby
<<<<<<< HEAD:app/controllers/votes_controller.rb
    @observation = naming.observation
    value = params[:value].to_i
    @observation.change_vote(naming, value)
    # redirect_with_query(
    #   controller: :observations,
    #   action: :show,
    #   id: @observation.id
    # )
    redirect_to observation_path(@observation.id, q: get_query_param)
=======
    observation = naming.observation
    observation.change_vote(naming, params[:value])
    redirect_with_query(controller: :observer,
                        action: :show_observation,
                        id: observation.id)
>>>>>>> master:app/controllers/vote_controller.rb
```

Reply:
@JoeCohen Looks like you already did this fix. This looks OK to merge, to me.

Reply:
I don't think this fix has actually been made, even though GitHub implies that it has.
I probably screwed something up.
I'm going to try again. Sorry if that causes additional emails.